### ISSUE TITLE
mtv.reconstrct(in main.py) method takes only 3 arguments, 4 were given.

### ISSUE THREAD

**Issue by MadanManav** [huybery/MvDSCN#2]
mtv.reconstruct function defined in model/rgbd,py takes only 3 arguments i.e. def reconstruct(self, view1, view2, lr) but in main.py when loss is being calculated in pre-train phase, there 3 views and lr given i.e loss = mtv.reconstruct(views[0], views[1], views[2], lr).

**Comment by huybery** [comment]
Thank you for your attention. I have deleted the redundant arguments.
